cli/tests/data/experimental/cassandra/config.json

```diff
@@ -0,0 +1,612 @@
+{
```

Comment:
I would avoid calling these files `experimental/....`. It is going to make it much easier for us to promote it to the root namespace later. This of "experimental" as presentation problem.